File: 1_iniciante/javascript/1037.js
/**
 * @source Beecrowd
 * @problem 1037 - Interval
 * @subject Selection
 *
 * @author Letícia Cruz
 * @date 2020-10-31
 */

var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split("\n");

let v = parseFloat(lines.shift());

if (v >= 0 && v <= 25) {
    console.log("Intervalo [0,25]");
} else if (v > 25 && v <= 50) {
    console.log("Intervalo (25,50]");
} else if (v > 50 && v <= 75) {
    console.log("Intervalo (50,75]");
} else if (v > 75 && v <= 100) {
    console.log("Intervalo (75,100]");
} else {
    console.log("Fora de intervalo");
}

File: 1_iniciante/java/1037.java
/**
 * @source Beecrowd
 * @problem 1037 - Interval
 * @subject Selection
 * 
 * @author Letícia Cruz
 * @date 2025-04-22
 */

import java.io.IOException;
import java.util.Scanner;

class Main {
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        double valor = sc.nextDouble();

        if (valor >= 0 && valor <= 25) {
            System.out.println("Intervalo [0,25]");
        } else if (valor > 25 && valor <= 50) {
            System.out.println("Intervalo (25,50]");
        } else if (valor > 50 && valor <= 75) {
            System.out.println("Intervalo (50,75]");
        } else if (valor > 75 && valor <= 100) {
            System.out.println("Intervalo (75,100]");
        } else {
            System.out.println("Fora de intervalo");
        }
        sc.close();
    }
}
